openslides/core/static/js/core/site.js

```diff
@@ -355,10 +355,27 @@ angular.module('OpenSlidesApp.core.site', [
                                     alreadyConverted.push(stack);
                                     break;
                                 case "img":
+                                    // pixel a4 in 72dpi: 595 x 842 pixel
+                                    // TODO: proper way to calculate the space left on the page.
+                                    // This requires further information
+                                    var width = parseInt(element.getAttribute("width")),
+                                        height = parseInt(element.getAttribute("height"));
+
+                                    if (width > 590) {
```

Comment:
Please put the hard coded values into a variable, like so:
```javascript
var maxResolution = {
    width: 590,
    height: 842
}
if (width > maxResolution.width) {
    var scale = maxResolution.width / width
...
```

Of course you can rename `maxResolution` to something else.